home_work_8/index.js

```diff
@@ -0,0 +1,188 @@
+const validateValue = (value) => {
+  if (value === undefined || value === null) {
+    return;
+  }
+  return value;
+};
+const EMPTY_VALUE = "This value is not specified";
+
+//1 Створити масив, довжину та елементи якого задає користувач (через prompt).
+const testArray = [];
+
+for (let i = 1; i < 2; i++) {
+  const lengthOfArray = +prompt(
+    "Enter the length of the array as a number (from 1 to 10)"
+  );
+  if (isNaN(lengthOfArray) || lengthOfArray > 10) {
```

Comment:
What if the length will be 0?

Reply:
Then we will get an empty array.
OR please specify what do you mean?

Reply:
>Then we will get an empty array.
I thought about it, and its ok to keep it as it is